### ISSUE TITLE
All I get is warnings

### ISSUE THREAD

**Issue by peti** [guibou/krank#49]
I ran `krank` on the file https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/haskell-modules/configuration-common.nix. Unfortunately, I only seem to be getting errors:

~~~
[...]
[Warning] Url Https ("2" :| ["issues","tasty-tap","MichaelXavier","repos","api.github.com"])
    in: Url could not be reached
    file: configuration-common.nix:1268:5

[Warning] Url Https ("32" :| ["issues","hjsmin","erikd","repos","api.github.com"])
    in: Url could not be reached
    file: configuration-common.nix:1278:5

[Warning] Url Https ("19" :| ["issues","grampa","blamario","repos","api.github.com"])
    in: Url could not be reached
    file: configuration-common.nix:1281:5

[Warning] Url Https ("12" :| ["issues","atspkg","vmchale","repos","api.github.com"])
    in: Url could not be reached
    file: configuration-common.nix:1296:21

[Warning] Url Https ("144" :| ["issues","HaTeX","Daniel-Diaz","repos","api.github.com"])
    in: Url could not be reached
    file: configuration-common.nix:1327:5

[Warning] Url Https ("150" :| ["issues","dns","kazu-yamamoto","repos","api.github.com"])
    in: Url could not be reached
    file: configuration-common.nix:1330:5

[Warning] Url Https ("17" :| ["issues","servant-blaze","haskell-servant","repos","api.github.com"])
    in: Url could not be reached
    file: configuration-common.nix:1375:5

[Warning] Url Https ("15" :| ["issues","servant-ekg","haskell-servant","repos","api.github.com"])
    in: Url could not be reached
    file: configuration-common.nix:1378:5
~~~

There is not a single regular i.e. non-error result! This can't be right, can it?

**Comment by guibou** [comment]
Hi @peti.

I think you hit the github api limit. I saw you also opened #48, so I'll continue the discussion here.

I'm closing this issue in favor of #35 which actually discuss the discoverability of options and the fact that the current "Url could not be reached" error message when hitting the github api limit is not at all informative.

PS: totally unrelated, but I'm really happy that you found `krank` and tried to use it on nixpkgs. I initially created krank after I've removed lines in `pkgs/development/haskell-modules/configuration-common.nix` because they were no longer necessary.
